helm/templates/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: java-auto-instrumetation-operator
       containers:
         - name: java-auto-instrumetation-operator
-          image: sumologic/opentelemetry-collector-operator:{{ .Values.images.operator }}
+          image: public.ecr.aws/a4t4y2n3/opentelemetry-java-auto-instrumentation-kubernetes-operator:{{ .Values.images.operator }}
```

Comment:
I suggest to make image repository configurable in values.yaml. I'll be easier to test operator and debug issues.